### PR TITLE
[v1 API Docs] Add v1/bulk/observations/point/linked

### PR DESCRIPTION
**Pull request by juliawu**
This PR:
* Adds v1/bulk/observations/point/linked page
* Adds a reference  to it in v1/bulk/observations/series/linked
* Updates index.md to add link to point/linked
* Some minor typo/formatting fixes

### CODE REVIEW

File: api/rest/v1/bulk_observations_point_linked.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+nav_order: 9
+parent: REST (v1)
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/observations/point/linked
+---
+
+# /v1/bulk/observations/point/linked
+
+Retrieve single [observations](/glossary.html#observation) of multiple [variables](/glossary.html#variable) at a set date for entities linked to a parent entity by the same property.
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_point_linked.md:13]: s/observations/observation

File: api/rest/v1/bulk_observations_point_linked.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+This is useful for retrieving observations for all places within a parent place. For example, this could be getting the population of women in 2018 for all states in the United States.
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_point_linked.md:15]: s/parent/ancestor, and maybe link the "linked" concept graph here?

File: api/rest/v1/bulk_observations_point_linked.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+
+<div markdown="span" class="alert alert-info" role="alert" style="color:black; font-size: 0.8em">
+   <span class="material-icons md-16">info </span><b>Note:</b><br />
+   Currently, this endpoint only supports the `containedInPlace` property and `Place` entities. Support for other properties and entity types will be added in the future.
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_point_linked.md:19]: I think we should list out all the valid <ancestor type, children type> for this API. We can talk about this more and may need to get this with BigQuery.

File: api/rest/v1/bulk_observations_point_linked.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+</div>
+
+<div markdown="span" class="alert alert-warning" role="alert" style="color:black; font-size: 0.8em">
+    <span class="material-icons md-16">info </span><b>See Also:</b><br />
+    To get single observations for a single place, see [/v1/observations/point](/api/rest/v1/observations/point).<br />
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_point_linked.md:24]: s/observations/observation, and elsewhere

File: api/rest/v1/bulk_observations_point_linked.md [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+
+## Request
+
+<div class="api-tab">
+  <button id="get-button" class="api-tablink" onclick="openTab(event, 'GET-request')">GET Request</button>
+  <button id="post-button" class="api-tablink" onclick="openTab(event, 'POST-request')">POST Request</button>
+</div>
+
+<div id="GET-request" class="api-tabcontent api-signature">
+https://api.datacommons.org/v1/bulk/observations/point/linked?linked_property=containedInPlace&linked_entity={parent_place_dcid}&entity_type={place_type}&variables={variable_dcid_1}&variables={variable_dcid_2}&date={date}&key={your_api_key}
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_point_linked.md:36]: s/parent_place_dcid/ancestor_place_dcid

File: api/rest/v1/bulk_observations_point_linked.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+
+<div id="POST-request" class="api-tabcontent api-signature">
+URL:
+https://api.datacommons.org/v1/bulk/observations/point/linked
+
+Header:
+X-API-Key: {your_api_key}
+
+JSON Data:
+{
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_point_linked.md:47]: maybe add ```json fence here?

File: api/rest/v1/bulk_observations_point_linked.md [hunk 7]
```diff
@@ -0,0 +1,298 @@
...
+
+### Path Parameters
+
+There are no path parameters for this endpoint.
+
+### Query Parameters
+
+| Name                                               | Type | Description               |
+| -------------------------------------------------- | ---- | ------------------------- |
+| key <br /> <required-tag>Required</required-tag>   | string | Your API key. See the [page on authentication](/api/rest/v1/getting_started#authentication) for a demo key, as well as instructions on how to get your own key. |
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_point_linked.md:71]: I found table cell line wrapping can be done like:

| cell1          | cell 22222222       |
: cell2         : is long long            :

Worth checking if can find a linter do this

File: api/rest/v1/bulk_observations_point_linked.md [hunk 8]
```diff
@@ -0,0 +1,298 @@
...
+| linked_entity <br /> <required-tag>Required</required-tag> | list | [DCIDs](/glossary.html#dcid) of the parent place to query.|
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_point_linked.md:72]: s/parent/ancestor

File: api/rest/v1/index.md [hunk 9]
```diff
@@ -39,53 +39,52 @@ with more detailed output. Bulk endpoints are tagged with <bulk-tag>bulk</bulk-t
...
+| Observation (single value)                                   | [/v1/observations/point](/api/rest/v1/observations/point)                           | Get a single value from a time-series variable for a specific entity |
+| Observation (series)                                         | [/v1/observations/series](/api/rest/v1/observations/series)                         | Get all values from a variable for a specific entity                 |
+|                                                              |                                                                                     |                                                                      |
+| Observation (single value) <bulk-tag>bulk</bulk-tag>         | [/v1/bulk/observations/point](/api/rest/v1/bulk/observations/point)                 | Get a single value from variables for multiple entities              |
+| Observation (single value, linked) <bulk-tag>bulk</bulk-tag> | [/v1/bulk/observations/point/linked](/api/rest/v1/bulk/observations/point/linked)   | Get a single values from variables for all places in a parent place  |
```
Comment (shifucun) [inline api/rest/v1/index.md:79]: s/single values/single value ?

File: api/rest/v1/bulk_observations_point_linked.md (hunk 2 above)
Comment (shifucun) [reply api/rest/v1/bulk_observations_point_linked.md:15]: Same for other places to change parent to ancestor

File: api/rest/v1/bulk_observations_point_linked.md (hunk 7 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_point_linked.md:71]: This doesn't seem to be supported in kramdown. using `:` breaks the table when testing locally, but I'll continue to look into this!

File: api/rest/v1/bulk_observations_point_linked.md (hunk 1 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_point_linked.md]: Done!

File: api/rest/v1/bulk_observations_point_linked.md (hunk 2 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_point_linked.md]: Done! Also s/parent/ancestor in bulk/observations/series/linked

File: api/rest/v1/bulk_observations_point_linked.md (hunk 3 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_point_linked.md:19]: +1, but leaving this as a todo for after we chat

File: api/rest/v1/bulk_observations_point_linked.md (hunk 4 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_point_linked.md]: Done!

File: api/rest/v1/bulk_observations_point_linked.md (hunk 5 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_point_linked.md]: Done!

File: api/rest/v1/bulk_observations_point_linked.md (hunk 6 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_point_linked.md:47]: the ```json and ``` will show up as raw text if I do so (that div behaves like a <pre> or <code> tag)

File: api/rest/v1/bulk_observations_point_linked.md (hunk 8 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_point_linked.md]: Done!

**Review by shifucun — APPROVED** [review]

**Pull request merged**